lib/matplotlib/backends/backend_ps.py

```diff
@@ -205,15 +205,29 @@ def set_linewidth(self, linewidth, store=True):
             if store:
                 self.linewidth = linewidth
 
+    @staticmethod
+    def _linejoin_cmd(linejoin):
+        # Support for directly passing integer values is for backcompat.
+        linejoin = {'miter': 0, 'round': 1, 'bevel': 2, 0: 0, 1: 1, 2: 2}[
+            linejoin]
+        return f"{linejoin:d} setlinejoin\n"
+
     def set_linejoin(self, linejoin, store=True):
         if linejoin != self.linejoin:
```

Comment:
Previously, `linejoin` would be normalized before comparing it with `self.linejoin`.

Reply:
That's not really a problem; that just means that if you inconsistently call set_linecap with either strings or numbers (but the latter should never happen), you'll get benign additional calls to setlinecap in the resulting PS stream.
I could deprecate support for numbers to enforce consistency, but that seems a bit gratuitious.